### DISCUSSION TITLE
easy access occ commands in docker

### DISCUSSION THREAD

**Discussion by ferdiga** [nextcloud/all-in-one#4947] [Wiki]
- create a script occ_docker and put it into an accessible bin directory + chmod +x
- put the following code into occ_docker 
```
#!/bin/sh
sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ "$@"
```
caveat
variables with blanks inside must be escaped with \\'

all occ commands are now easily available

**Comment by Zoey2936** [comment]
I think it could be easier, like this:
```bash
#!/bin/sh
sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ "$@"
```
